### DISCUSSION TITLE
v5.0.0-RC1

### DISCUSSION THREAD

**Discussion by michaelliunsky** [michaelliunsky/niRvana-theme#31] [General]
**更新后需要将所有已经存在的页面进入编辑器点一下更新，否则无法访问页面。**

- **题外话**：**在除夕夜发布的 RC 版本预示着 v5.0 开发迈入新台阶。祝各位龙行龘龘，前程朤朤。龙年快乐！**
- **删除**：自动添加页面 HTML 后缀功能。**注意**因此导致**更新后需要将所有已经存在的页面进入编辑器点一下更新，否则无法访问页面。**
- **性能优化**：更改了 Gravatar 为更快镜像源
- **更新**：fontawesome 版本
- **更新**：**全新适配 PHP8.3**

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelliunsky/niRvana-theme/releases/tag/v5.0.0-RC1'>v5.0.0-RC1</a>.</em>

**Comment by ebjwly** [comment]
您的网站怎么也打不开了？

**Reply by michaelliunsky** [reply]
已恢复了
